iaso/urls.py

```diff
@@ -125,7 +125,7 @@
 router.register(r"entitytype", EntityTypeViewSet, basename="entitytype")
 # At the moment we use the same view set but separate it for the future for when we want to be able to
 # change the format in the future
-router.register(r"mobile/entitytype", EntityTypeViewSet, basename="entitytype")
+router.register(r"mobile/entitytype", MobileEntityTypesViewSet, basename="entitytype")
```

Comment:
Dont we use plurals name now ? 
I added mobile/entitytypes", MobileEntityTypesViewSet, basename="entitytypes at line 124 in urls.py in my previous PR about mobile entity types. Indeed on this branch we have entitytype and entitypes working. So I think maybe something went wrong during your merge of main ? And/Or I forgot to remove the old url

Reply:
The old URLs are still there for backward compatibility reasons